Syntaxes/Makefile.plist

```diff
@@ -64,7 +64,7 @@
 						</dict>
 					</dict>
 					<key>end</key>
-					<string>\n</string>
+					<string>(?=[^\\])$</string>
```

Comment:
This and the similar change below don't actually do anything. The look-ahead in ensuring there is not a `\` as the next character while then ensuring it is the end of the line. Both are always going to be true.

Also we do want to match the `\n` here to more proper scope the caret when it is at the end of the line since that is the default current scope (to the right of the caret).

Reply:
You are right. I had a trouble to continue comment which was started inside prerequisites so I tried to continue until I find "[^\\]\n". But here is a mistake definitely (it does nothing as you said). I should fix and re-test.

Reply:
About current status. Unfortunately, when I return \n instead of (?=\\)$ different bad things happen. I try to find why, but unfortunately, I struggled a lot previously, and now I'm not sure now I'll find a solution instead quickly. For example:

.PHONY: all
all: prereq # <------- "all:" looses the highlighting, or prereqs loose colorizing, etc.

**Test commit**: https://github.com/fadeevab/make.tmbundle/commit/db317ee2873264b96ab6f234683bee3b6324a0f0#diff-a941b8105726ed2f6d1ddbb7e4c9e996
**Result:** https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Ffadeevab%2Fmake.tmbundle%2Fdb317ee2873264b96ab6f234683bee3b6324a0f0%2FSyntaxes%2FMakefile.plist&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2Ffadeevab%2Fvscode%2Fmaster%2Fextensions%2Fmake%2Ftest%2Fcolorize-fixtures%2Fmakefile&code=